src/ubx.cpp

```diff
@@ -108,10 +108,12 @@ GPSDriverUBX::configure(unsigned &baudrate, OutputMode output_mode)
 {
 	_configured = false;
 	_output_mode = output_mode;
+#if !defined(__PX4_POSIX_RPI)
```

Comment:
It would be good if you could change this to an interface flag instead ```configure(enum interface, unsigned baud, output mode)```. Then the interface setting controls wether or not the baud rate is applied